root/static/scripts/edit/externalLinks.js

```diff
@@ -217,6 +226,33 @@ export class _ExternalLinksEditor
       );
     });
 
+    const sourceId = isDatabaseRowId(sourceData.id) ? sourceData.id : 'new';
+    const submittedLinksKey = `submittedLinks_${sourceType}_${sourceId}`;
+    this.submittedLinksWrapper = {
+      get() {
+        if (hasSessionStorage) {
```

Comment:
There's no need to check `hasSessionStorage` before calling `sessionStorageWrapper` methods, because they're just no-ops in that case. The reason I'm checking it is to avoid other expensive calls (`compactEntityJson`, `decompactEntityJson`).